cnxarchive/sql/search/title.part.sql

```diff
@@ -10,4 +10,4 @@ SELECT
 FROM
   latest_modules
 WHERE
-  name ~* req(%({0})s::text)
+  StripHTML(name) ~* req(%({0})s::text)
```

Comment:
Creating an index should be very simple, see https://github.com/Connexions/cnx-archive/blob/master/cnxarchive/sql/schema/indexes.sql:

```
CREATE INDEX modules_strip_html_name ON modules (strip_html(name));

CREATE INDEX latest_modules_strip_html_name ON latest_modules (strip_html(name));
```

(Also, let's rename `StripHTML` to `strip_html`, I think we've been using lower case with underscore function names.)